assets/infrastructure/pingo.json

```diff
@@ -0,0 +1,28 @@
+{
+  "metadata": {
+      "label": "pingo",
+      "category": "infrastructure",
```

Comment:
@ohld is it correct tag?

Reply:
Definitely not. The description of infrastructure category is "TON Blockchain infrastructure contracts". 

I'm not sure what will be the best category here. I guess the closest one is "gaming". But let's think about the better category for a class of projects like this. And some other examples that might have the same category /cc @Caranell

Reply:
Well I wouln't also call it gaming exactly, they're _pretending_ to be working on some DePIN (https://pingo-work.gitbook.io/pingo/product/pingo-cdn-service) and even have an app to run on your linux 
<img width="962" alt="Screenshot 2025-03-18 at 13 48 48" src="https://github.com/user-attachments/assets/416384c0-4638-435c-9c6d-19417f735e04" />
Without any real use atm we might add it to gaming, like memhash, but I couln't say for sure if they're working on some real DePIN (in that case we could add a new category)